src/c2pa/c2pa.py

```diff
@@ -1427,18 +1612,25 @@ def wrapped_callback(
                 signed_bytes_ptr,
                 signed_len):
             # Returns -1 on error as it is what the native code expects.
-            # The reason is that otherwise we ping-pong errors between native code and Python code,
-            # which can become tedious in handling. So we let the native code deal with it and
+            # The reason is that otherwise we ping-pong errors
+            # between native code and Python code,
+            # which can become tedious in handling.
+            # So we let the native code deal with it and
             # raise the errors accordingly, since it already does checks.
             try:
-                if not data_ptr or data_len <= 0 or not signed_bytes_ptr or signed_len <= 0:
+                if (
+                    not data_ptr
+                    or data_len <= 0
+                    or not signed_bytes_ptr
+                    or signed_len <= 0
+                ):
                     # Error: invalid input, invalid so return -1,
                     # native code will handle it!
-                    return -1
+                    return -1  # pragma: no cover
```

Comment:
We probably should have test cases for these, so this will just hide that they are missing. But it may not be practical to create them. I'm not sure what the goal is here.